source/DataWriter.h

```diff
@@ -82,7 +83,7 @@ class DataWriter {
 
 private:
 	// Save path (in UTF-8). Empty string for in-memory DataWriter.
-	std::string path;
+	std::filesystem::path path = std::filesystem::path();
```

Comment:
```suggestion
	std::filesystem::path path = {};
```
shorter this way

Reply:
```suggestion
	std::filesystem::path path();
```
this is even shorter :smile:

Reply:
or you know just leave it unitialized, that should default to nothing